change/@splunk-otel-617b13c9-328a-4e4b-bd44-42a157069ae0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Update development dependencies",
+  "packageName": "@splunk/otel",
+  "email": "Rauno56@gmail.com",
+  "dependentChangeType": "patch"
```

Comment:
I don't think we should add changelog entry for internal stuff.

Reply:
cc: @jtmalinowski

Reply:
Merging it, since we can later decide whether to include those or not. The current agreement is to create the change files.